### ISSUE TITLE
everpad crash: TypeError: find_notes()

### ISSUE THREAD

**Issue by hdave** [nvbn/everpad#21]
Just upgraded my Ubuntu 12.04 via PPA this morning. On this new version, everpad itself is crashing:

```
> cat /var/crash/_usr_bin_everpad.1000.crash

...
Traceback:
 Traceback (most recent call last):
   File "/usr/lib/pymodules/python2.7/everpad/pad/indicator.py", line 42, in update
     20, Note.ORDER_UPDATED_DESC,
   File "/usr/lib/python2.7/dist-packages/dbus/proxies.py", line 145, in __call__
     **keywords)
   File "/usr/lib/python2.7/dist-packages/dbus/connection.py", line 651, in call_blocking
     message, timeout)
 DBusException: org.freedesktop.DBus.Python.TypeError: Traceback (most recent call last):
   File "/usr/lib/python2.7/dist-packages/dbus/service.py", line 707, in _message_cb
     retval = candidate_method(self, *args, **keywords)
 TypeError: find_notes() takes at most 6 arguments (7 given)
...
```

Hope this helps.

**Comment by nvbn** [comment]
It's after upgrade and reboot?

**Comment by hdave** [comment]
Yes.  After upgrade and reboot.

**Comment by nvbn** [comment]
Wait https://launchpad.net/~nvbn-rm/+archive/ppa/+builds?build_text=&build_state=pending
Fixed packages sended to launchpad day ago.

**Comment by hdave** [comment]
Build failed :-(

https://launchpad.net/~nvbn-rm/+archive/ppa?field.series_filter=precise

**Comment by nvbn** [comment]
I push packages to retry building, but some shit happening with launchpad.

**Comment by hdave** [comment]
fixed in 1.4.0-0
